sqls/test3.sql

```diff
@@ -0,0 +1 @@
+SELECT * FROM test;
```

Comment:
🚫 **[sqlfluff]** <[AM04](https://docs.sqlfluff.com/en/stable/rules.html#rule-AM04)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[ambiguous.column_count] - Query produces an unknown number of result columns.
<!-- __reviewdog__:ChBlNmU2OTEyZjMyNjNiZjFiEghzcWxmbHVmZg== -->